data/json/uncraft/generic.json

```diff
@@ -33,7 +33,7 @@
     "difficulty": 1,
     "time": 2200,
     "qualities": [ { "id": "CUT", "level": 1 } ],
-    "components": [ [ [ "rag", 60 ] ], [ [ "sheet", 4 ] ] ]
+    "components": [ [ [ "rag", 60 ] ], [ [ "sheet", 4 ] ], [ [ "scrap", 100 ] ] ]
```

Comment:
I'd suggest reducing the scrap a lot, and adding a couple units of wire instead. Maybe 20 scrap and 20 wire? I'm going with the idea that an innerspring could basically be used to make a segment of chain link fence (in the most redneck way imaginable), and it takes 20 wire to make a segment of chainlink fence.